File: app/src/main/java/com/vikingsen/cheesedemo/model/webservice/dto/CheeseDto.java
package com.vikingsen.cheesedemo.model.webservice.dto;


import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class CheeseDto {
    private long id;
    private String name;
    private String image;
    private String description;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/vikingsen/cheesedemo/model/webservice/dto/CheeseDto.kt
package com.vikingsen.cheesedemo.model.webservice.dto


import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class CheeseDto(val id: Long, val name: String, val image: String, val description: String, val sort: Int)
